src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -56,6 +57,7 @@ public partial class NodeViewModel : ViewModelBase
         private bool isexplictFrozen;
         private bool canToggleFrozen = true;
         private bool isRenamed = false;
+        private ObservableCollection<MenuItem> dismissedAlerts = new ObservableCollection<MenuItem>();
```

Comment:
hmm.. I think it is usually not recommended to referencing or dynamically manage view level controls in view model. Is there a workaround of this, using binding for example so we can keep the `ObservableCollection<MenuItem> dismissedAlerts` on node view level but bind to a collection of messages on nodeviewModel maybe?